.github/workflows/release.yml

```diff
@@ -44,3 +53,4 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           artifacts: /home/runner/work/blueflood/blueflood/blueflood-all/target/blueflood-all-${{ github.event.inputs.version }}-jar-with-dependencies.jar
           tag: "blueflood-${{ github.event.inputs.version }}"
+          body: "${{ steps.changelog.outputs.changelog }}"
```

Comment:
Something here doesn't look right. Either the changelog file will always have only the latest changes in it every time, or every release will have the entire history of the project in it. The changelog file and release body should be treated differently.

Reply:
Changelog file does have the latest changes in it. What do we want in release body?

Reply:
Only the changes for that release, right?

Reply:
which means all the commits since last release?

Reply:
Well, with the way we're going now, yes. What's important is that a release notifies users what's new in the release.

Reply:
So I tested it on my fork and release body does have the commits list since last release.

Reply:
So do you want to go ahead and merge this like this and then figure out where to go from there?

Reply:
Sure... I will figure out a way.